### DISCUSSION TITLE
IKEA Symfonisk - battery drain or wrong value reported

### DISCUSSION THREAD

**Discussion by michal-bartak** [Koenkk/zigbee2mqtt#8573] [Devices]
About month ago I started with Symphonisk connected to my HA through zigbee2mqtt using Conbee II.
After a month I can see both devices on 5% and 34% respectively. 
Seeing that I upgraded fw of my ConbeeII to most recent one (I had one from last Autumn)

Today morning one of those controllers dropped to 0%. But when I touched it, the battery jumped to 60%. An hour later it shows 21%. I suppose I'm facing generally known issue with IKEA remotes. It's strange however that the device still works at 0% battery and jumps to 60%.

Interesting thing happened to me at beginning of playing with those controllers. Once I turned one to the left, it started to infinitely generating rotate left message. I saw it in NodeRed. It stopped after I touched the controller again. Hard to say it's faulty controller or but in software in-between. Never happened again. I suppose it could empty battery in a day.

Anyway, is there known solution to battery issues with IKEA remotes? I've read a several threads in internet which blames all other routers but IKEA ones. Others write that they resolved it with ConbeeII. But all those information is pretty scattered without final judgment. Considering IKEA symfonisk is whitelabel of Sonof... is Sonof out of the game too?

**Comment by MattWestb** [comment]
All IKEA controllers is only updating the battery level then you using the button and "between" they is sending the same level as the last red one (can sending the same for mounts if not being used as my in the bathroom). 
Also the battery level is very "jumpy" not smooths as Xiaomi sensors that is having very nice curve in Grafana.
Normally the best is looking  if the pairing LED is blinking then using the controller then it battery is "on the last mile" and if its being complete drained its very likely you must repairing in then it can being its have corrupted the NVM and the network security counters is wrong and cant communicating (is one bug in Silabs Zigbee stack that is fixed in the last releases).

Most problem was / is with Somoff ZBB that have bad hardware and wrong setting is the firmware but its can happening with other coordinators that is not playing nice with sleeping end device so best connecting them to good routers and not the coordinator for not getting problems.

IKEA Zigbee devices is **not** made or sold by Sonoff then they is making there own hard and software and IKEA is selling some million more devices / year and is updating there firmware then having problems and is certifying all there devices that most of the Chinese companies not doing.
